src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -645,15 +650,22 @@ export default class FilingHistoryList extends Vue {
         )
       }
 
+      // add properties for Consent to Continuation Outs
+      if (this.isTypeConsentContinuationOut(filing)) {
+        // FUTURE: expiry date may come from business object
+        item.expiry = this.apiToPacificDateTime(filing.data.consentContinuationOut?.expiry)
+        item.details = filing.data.consentContinuationOut?.orderDetails
+      }
+
       // add properties for staff filings
       if (this.isTypeStaff(filing)) {
         // court orders may have documents so leave property as null (they load on toggle)
         if (!this.isTypeCourtOrder(filing)) {
           item.documents = [] // no documents
         }
-        item.fileNumber = filing.data.order?.fileNumber || '' // may be falsy
+        item.fileNumber = filing.data.order?.fileNumber || '' // may be absent
         item.isTypeStaff = true
-        item.notationOrOrder = filing.data.order?.orderDetails // should not be falsy
+        item.details = filing.data.order?.orderDetails // should always be present
```

Comment:
We don't need 3 completely different names for the same thing. (It's called `details` when we create a staff notation filing. It's called `orderDetails` in the API response. And it _was_ called `notationOrOrder` here.)

Reply:
Ha, good catch. Naming things is SO difficult.